internal/testdata/basic.proto3test.yaml

```diff
@@ -131,3 +131,7 @@ values:
   - single_int32_wrapper: 1
   - single_int32_wrapper:
       value: 1
+  - single_any:
+        "@type": "type.googleapis.com/google.protobuf.Any"
+        type_url: "type.googleapis.com/google.protobuf.BoolValue"
+        value: ""
```

Comment:
This does not match the format that `protojson` uses. Also, why is `value` empty here? Is it expected to be base64-encoded bytes, and it happens to be an empty message?

The format that `protojson` uses is what was mentioned in #20:

```json
{
	"@type": "google.protobuf.Any",
	"value": {
		"@type": "google.protobuf.FileDescriptorProto",
		"name": "foo/bar/baz.proto"
	}
}
```

So that would warrant a slightly different implementation change.

Reply:
Very true (I am going to claim brain fog). Updated.